src/test/moment/zones.js

```diff
@@ -461,9 +461,27 @@ test('timezone format', function (assert) {
 
 test('parse zone without a timezone', function (assert) {
     test.expectedDeprecations();
-    var m = moment.parseZone('2016-02-01T00:00:00');
+    var m1 = moment.parseZone('2016-02-01T00:00:00');
+    var m2 = moment.parseZone('2016-02-01T00:00:00Z');
+    var m3 = moment.parseZone('2016-02-01T00:00:00+00:00'); //Someone might argue this is not necessary, you could even argue that is wrong being here.
+    var m4 = moment.parseZone('2016-02-01T00:00:00+0000'); //Someone might argue this is not necessary, you could even argue that is wrong being here.
     assert.equal(
-        m.format('M D YYYY HH:mm:ss ZZ'),
+        m1.format('M D YYYY HH:mm:ss ZZ'),
+        '2 1 2016 00:00:00 +0000',
+        'Not providing a timezone should keep the time and change the zone to 0'
```

Comment:
Ok, so discussing offline with the moment core team, this one is fine.  It's not because we're aligning to ISO8601 defaults, but because the `parseZone` function is designed to return a fixed-offset moment - which doesn't work for local time.  If no zone offset is present, then it's the same as if you did `moment("2016-01-01T00:00:00").utcOffset("", true)`, and since `"" == 0` in JavaScript truth table, then it's the same as setting a zero offset.